src/main/java/GameSelector.java

```diff
@@ -16,8 +16,17 @@ Game gameSelection(int gameType) {
                 playerTwo = new RandomComputer();
                 break;
             case 3:
+                playerOne = new Human();
+                playerTwo = new MiniMaxComputer();
+                break;
+            case 4:
```

Comment:
i know this PR is for minimax, but lets use Enums here instead of magic numbers so that it reads better. something like this:

```
case HumanVsMiniMax:
  playerOne = new Human();
  playerTwo = new MiniMaxComputer();
  break;
```

Reply:
^ same for all cases

Reply:
enums in place